pyproject.toml

```diff
@@ -2,9 +2,97 @@
 requires = ["pip==21.3.1", "setuptools==59.6.0", "setuptools-scm[toml]>=6.2"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "pymanopt"
+description = "Toolbox for optimization on Riemannian manifolds with support for automatic differentiation"
+authors = [
+  {name = "Jamie Townsend"},
+  {name = "Niklas Koep"},
+  {name = "Sebastian Weichwald"},
+]
+readme="README.md"
```

Comment:
```suggestion
readme = "README.md"
```

Reply:
Addressed in edda1b20cecb5a0b2ee767471ba284f301713545